server/cmt_provision.go

```
@@ -0,0 +1,597 @@
+			"ref":    branch,
+			"inputs": workflowInputs,
+		})
+	if err != nil {
+		return 0, fmt.Errorf("failed to create CMT workflow dispatch request: %w", err)
+	}
+
+	resp, err := client.Do(dispatchCtx, req, nil)
+	if err != nil {
+		return 0, fmt.Errorf("failed to dispatch compatibility-matrix-testing.yml: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)